### ISSUE TITLE
Some flags/options lost after auto-reconnecting

### ISSUE THREAD

**Issue by tha-G** [FreeRDP/FreeRDP#7485]
After the connection is lost freerdp succeeds in auto-reconnecting, most of the time without me actually noticing it, wonderful.

## But after this automatic reconnect, some flags are lost. The clearly noticeable ones being +clipboard -wallpaper -themes -decorations -fonts.

I use the following command line to connect:
"/opt/freerdp-nightly/bin/xfreerdp /home/some_name.rdp /gfx:AVC444 /f /bpp:24 +auto-reconnect +multimon +clipboard -wallpaper -themes -decorations -fonts"

I connect via a gateway, the content of my rdp file looks like this:
"gatewayhostname:s:xxxx
gatewayusagemethod:i:1
gatewaycredentialssource:i:4
gatewayprofileusagemethod:i:1
gatewaybrokeringtype:i:0

full address:s:xxxx
username:s:xxxx

prompt for credentials:i:0
promptcredentialonce:i:1"

System connecting to: Windows 10 Enterprise 64-bit
Operating system: Ubuntu 18.04.6 LTS 64-bit
FreeRDP version: nightly 3.0.0-dev (1265114be) (but have seen the same behavior with the default version 2.2.0 on my system)

**Comment by akallabeth** [comment]
Do you mean that your `clipboard` no longer works?
that is not the case here with my test setup, so maybe some information to reproduce is missing?

**Comment by tha-G** [comment]
Hi akallabeth,

Yes indeed, but it's hard to pinpoint. It seems sometimes some flags/options are kept and sometimes some do not.

For example just now I was working remotely and tested clipboard every few minutes. It kept working so to force an auto-reconnect (disconnect and reconnect my network cable)

1th reconnect, clipboard still working, but **wallpaper showed up** (which should be disabled because of the -wallpaper flag)
2nd reconnect, clipboard still working, but **wallpaper disappeared** again
3rd reconnect, clipboard still working, but **wallpaper showed up** again
4th reconnect, clipboard still working, but **wallpaper disappeared** again

I regularly notice the clipboard to stop working but just now I could not reproduce it too.
In a more than 2 hours session I had a small amount of (non forced) reconnects which again toggled the wallpaper as shown or not

Since I found this wallpaper toggling strange I also closed the session and reopened the session a few times (quit and rerun command)
Again, it was alternately showing the wallpaper and hiding it...?

So I guess the flags/options are not lost after auto-reconnecting but can not be enforced every time? The consistent toggling of the wallpaper is suspicious.
Are you able to reproduce this?

I'm happy to test things out to help, thanks!
